content/docs/04.5-devx/05-app-profile/02-container-deployment.md

```diff
@@ -58,6 +71,20 @@ To add a container tier to Palette Dev Engine App Profile,
        argument: HOSTNAME
  
  
-4. **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
+6. Click **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
+
```

Comment:
The next step should be to deploy the App Profile through the App page. In the App page, the user would deploy a new app by using the App Profile.

![CleanShot 2022-11-28 at 07 34 53](https://user-images.githubusercontent.com/29551334/204304145-d968acc9-9733-4d38-9b58-befd5a5175ab.png)